tests/test_amity.py

```diff
@@ -9,34 +10,37 @@ class TestAmity(unittest.TestCase):
 	def setUp(self):
 		self.amity = Amity()
 	
-	@patch('app.storage')	
-	def test_room_creation(self,test_storage):
-		test_storage.list_of_all_rooms.return_value = ["CAMELOT","NARNIA"]
-		result = Mock()
-		result.room_name.return_value = "HOGWARDS"
-		print test_storage.list_of_all_rooms.return_value
-		self.result = self.amity.createRoom(result.room_name.return_value)
-		print self.amity.room_name
-		print test_storage.list_of_all_rooms.return_value
-
-		if self.result != "Room created successfully!!":
-			self.assertNotEqual(
-				self.amity.room_name, "", msg = "Cannot create empty room")
-
-			self.assertNotIn(
-				self.amity.room_name,
-				test_storage.list_of_all_rooms.return_value,
-				msg = "Room with that name already exist."
-				)
-			print test_storage.list_of_all_rooms.return_value
+	def test_room_creation(self):
+		size = len(Storage.list_of_all_rooms)
+		if self.amity.createRoom("Hogwards") == "Room created successfully!!":
```

Comment:
Test seems to be skipping some test. It would be better to test if your room creating function works instead of evading it with an if. Something like:

```
room_created_status = self.amity.createRoom("Hogwards")
self.assertEqual(room_created_status, "Room created successfully!!")
self.assertEqual(len(Storage.list_of_all_rooms), size+1)
```